hack/cspell-ignore.txt

```diff
@@ -10,6 +10,7 @@ Rsvd
 WSREP
```

Comment:
Should we add E2E or e2e or E2e? I have seen these 3 terms.

Reply:
all seem to be used inter-changebly. but good idea to follow a convention.